### PR TITLE
Adds basic html escaping for websockets example

### PR DESCRIPTION
**Pull request by jkotalik**
There is some basic XSS security issues with the sample code. I went ahead and added a basic (not perfect) fix for it. Also maybe adding a disclaimer should be good.

### CODE REVIEW

File: aspnetcore/fundamentals/websockets/sample/wwwroot/index.html [hunk 1]
```diff
@@ -39,6 +39,7 @@
         </tbody>
     </table>
 
+<script src="jquery-3.2.1.min.js"></script>
```
Comment (anurse) [inline aspnetcore/fundamentals/websockets/sample/wwwroot/index.html]: Is this used?

Comment (jkotalik) [inline aspnetcore/fundamentals/websockets/sample/wwwroot/index.html]: I was originally going to use jQuery for escaping, I'll remove it.

**Comment by jkotalik** [comment]
@anurse :up: :date:

**Comment by jkotalik** [comment]
#

**Comment by anurse** [comment]
https://github.com/aspnet/Docs/commit/d182d04934c2b35cc61de8ff00b212914965ad1c

**Pull request closed without merging**